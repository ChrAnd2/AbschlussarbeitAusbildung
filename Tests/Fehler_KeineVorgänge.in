//******************************************
//+ Fehler_KeineVorgänge - keine Daten
//******************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
