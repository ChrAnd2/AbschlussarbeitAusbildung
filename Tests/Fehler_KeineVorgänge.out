//******************************************
//+ Fehler_KeineVorgänge - keine Daten
//******************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger


*************************************
Es ist ein Fehler aufgetreten.
*************************************
EingabeException: Die Datei enthält keine Vorgänge.
